### PR TITLE
Show baggage info

### LINKED ISSUE #453: Add possibility to show baggage info
Acceptance criteria:

- [x] When clicking on Baggage, specific FAQ category is shown
- [x] List of bought baggages is shown
- [x] "Add more baggage" redirects Baggage section in MMB
- [ ] Show info that no adiditional baggage can be added if that happens

screen shot 2018-06-20 at 12 00 01

Zeplin: https://zpl.io/bPYv4nk

### PR DESCRIPTION
**Pull request by jalbertsr**
Close #453 

Backend says api is a bit outdated and some baggage types are buggy so this version only has: 
 - Cabin baggage
 - Checked baggage

Missing:
- Personal Items
- Possibility to know passanger name associated to one exact baggage. 
- Extra and overweight baggage.
<br/><br/><br/><url>LiveURL: https://smartfaq-show-baggage-info.surge.sh</url>

### CODE REVIEW

**Comment by kantord** [comment]
Still WIP?

**Comment by jalbertsr** [comment]
yes, is reday to CR

**Comment by kantord** [comment]
2018-06-28-143542_2362x1899_scrot

something is not kosher on Firefox

File: src/context/ExtraInfoState.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      isBaggageVisible: !prevState.isBaggageVisible,
+    }));
+  };
+
+  render() {
+    const { isBaggageVisible } = this.state;
+
+    return (
+      <ExtraInfoState.Provider
+        value={{ isBaggageVisible, toggleBaggage: this.toggleBaggage }}
```
Comment (jaroslav-kubicek) [inline src/context/ExtraInfoState.js]: FYI: there is one caveat when using React context, see this: https://reactjs.org/docs/context.html#caveats I'm already fixing it in #494

**Pull request merged**